### PR TITLE
GCE token auth support

### PR DESCRIPTION
**Pull request by robertabbott**
Testing: encrypted an image
then updated it

### CODE REVIEW

File: brkt_cli/update_encrypted_gce_image_args.py [hunk 1]
```diff
@@ -15,6 +15,20 @@ def setup_update_gce_image_args(parser):
         required=False
     )
     parser.add_argument(
+         '--brkt-email',
+         metavar='EMAIL',
+         dest='api_email',
+         help='Bracket user email',
+         required=True
```
Comment (krishnanrs) [inline brkt_cli/update_encrypted_gce_image_args.py]: Do these arguments need to be mandatory here, especially if this was already passed when originally encrypting the image?

**Comment by krishnanrs** [comment]
Except for that one comment and my reservations about using user credentials while authenticating, the changes looks good to me.

File: brkt_cli/update_encrypted_gce_image_args.py (hunk 1 above)
Comment (robertabbott) [inline brkt_cli/update_encrypted_gce_image_args.py]: fixed

**Comment by robertabbott** [comment]
for the time being user credentials is what we have to use. This may change in the future

**Comment by krishnanrs** [comment]
lgtm, except for the test failures now. grant explained that the user credentials are temporary until we have the tagging feature in place.

**Pull request merged**